ci/tasks/upgrade-stemcell.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+dir=`dirname "$0"`
+source ${dir}/utils.sh
+
+base=$( cd "$( dirname "$( dirname "$0" )")"/.. && pwd )
+base_gopath=$( cd $base/../../../.. && pwd )
+go version
+export GOPATH=$base_gopath:$GOPATH
+echo "GOPATH=" $GOPATH
+
+print_title "INSTALL BOSH CLI..."
+echo "installing bosh CLI"
+gem install bosh_cli --no-ri --no-rdo c
```

Comment:
I would like to have a try with new BOSH CLI, but not sure if we can switch to it at this stage, maybe after I got familiar with params and options, we can start using new CLI in our pipeline.